src/uniformscalingmap.jl

```diff
@@ -70,6 +70,14 @@ for (In, Out) in ((AbstractVector, AbstractVecOrMat), (AbstractMatrix, AbstractM
     end
 end
 
+function _unsafe_mul!(M::AbstractMatrix, L::UniformScalingMap, s::Number, a::Number, b::Number)
+    rmul!(M,b)
+    for (i,j) in zip(axes(L)...)
+        M[i,j] += L.λ * s * a
```

Comment:
Probably this gets hoisted out of the loop, but we could as well make sure manually that the rhs gets computed only once.